docs/fsharp/index.md

```diff
@@ -44,15 +44,23 @@ To learn more and get involved, check out [fsharp.org](http://fsharp.org).
 
 ## Documentation
 
-* [Getting Started](tutorials/getting-started/index.md)
-* [Introduction to Functional Programming](introduction-to-functional-programming/index.md)
 * [Tutorials](tutorials/index.md)
+* [Introduction to Functional Programming](introduction-to-functional-programming/index.md)
 * [Language Reference](language-reference/index.md)
 * [F# Core Library Reference](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/fsharp-core-library-reference)
 
+## Free Learning Resources
```

Comment:
Why emphasize "free" here? Am I supposed to think that "Video Learning Resources" and "Further Resources" are either not gratis or not libre?